doc/content/verification_and_validation/val-2a.md

```diff
@@ -1,58 +1,59 @@
 \end{equation}
 
-where $C$ is the concentration of deuterium in PCA, $t$ is the time, $x$ is the distance from the upstream side, $D$ is the diffusivity of deuterium in PCA, and $S$ is the source term in PCA due to the deuterium ion beam.
+where $C$ is the concentration of deuterium in PCA, $t$ is the time, $D$ is the diffusivity of deuterium in PCA, and $S$ is the source term in PCA due to the deuterium ion implantation.
 
-Second, the deuterium become gas on the two sides of PCA during recommendation process, which is described as:
+Second, the deuterium recombines into gas on both sides of the PCA sample. By assuming that the recombination process is at steady state (which is not a necessary assumption in TMAP8, but appropriate in this case), it is described as the following surface flux:
 
 \begin{equation} \label{eq:recommendation}
-J = A (K_r C^2 - K_d P^2),
+J = A (K_r C^2 - K_d P),
 \end{equation}
 
-where $J$ is the recommendation flux out of the upstream or downstream side, $A$ is the area on the upstream or downstream side, $P$ is the pressure on corresponding side, and $K_r$ and $K_d$ are the recommendation coefficient and dissociation coefficient, respectively.
+where $J$ is the recombination flux out of the sample sides, $A$ is the area on the upstream or downstream side, $P$ is the pressure on the corresponding side, and $K_r$ and $K_d$ are the recombination and dissociation coefficients, respectively.
 
-The objective of the model is to determine the permeate flux on the downstream side and match the experimental data published in [!cite](anderl1985tritium) and reproduced in [val-2a_comparison].
+The objective of this simulation is to determine the permeation flux on the downstream side and match the experimental data published in [!cite](anderl1985tritium) and reproduced in [val-2a_comparison].
 
 ## Model Description
 
-In this case, TMAP8 simulates a one-dimensional domain to represent the deuterium diffusion. The deuterium is generated by ion beam and leave from upstream and downstream side by recommendation. Note that this case can easily be extended to a two- or three-dimensional case.
+In this case, TMAP8 simulates a one-dimensional domain to represent the deuterium implantation, diffusion, and recombination. Note that this case can easily be extended to a two- or three-dimensional case.
 
-The source term in the model is described as a normal distribution instead of a piecewise function from TMAP4. The source term of deuterium versus the coordinate is defined as:
+The source term in the model is described as a normal distribution instead of the piecewise function from TMAP4 [!citep](longhurst1992verification). The source term of deuterium from ion beam implantation is defined as:
 
 \begin{equation} \label{eq:normal_distribution}
-S = \frac{1.5}{\sigma \sqrt(2 \pi) \exp \left( -0.5 * ((x - \mu ) / \sigma)^2 \right)},
+S = F \frac{1.5}{\sigma \sqrt(2 \pi)} \exp \left( - \frac{(x - \mu )^2}{2 \sigma^2} \right),
```

Comment:
Why the 1.5 factor in the nominator? Maybe explain in the text below.

Reply:
The factor of 1.5 is from TMAP4, I will explain it on documentation.